Index: src/test/java/com/encore/bbs/controller/BbsControllerTest.java
===================================================================
diff --git a/src/test/java/com/encore/bbs/controller/BbsControllerTest.java b/src/test/java/com/encore/bbs/controller/BbsControllerTest.java
deleted file mode 100644
--- a/src/test/java/com/encore/bbs/controller/BbsControllerTest.java	
+++ /dev/null	
@@ -1,42 +0,0 @@
-package com.encore.bbs.controller;
-
-
-import com.encore.bbs.board.controller.BbsController;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
-import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.setup.MockMvcBuilders;
-
-import static java.lang.reflect.Array.get;
-
-//@SpringBootTest
-//@ExtendWith(SpringExtension.class)
-//class BbsControllerTest {
-//
-//    @Autowired
-//    BbsControllerTest bbsController;
-//
-//    public MockMvc mockMvc; //api 테스트 mockmvc
-//    /* Spring Boot와 Spring Framework의 최신 버전에서는 @SpringBootTest와 @ExtendWith(SpringExtension.class)가 사용되면
-//     테스트 컨텍스트가 올바르게 설정되어 MockMvc 인스턴스를 자동으로 생성하므로, @Autowired를 사용하여 mockMvc 필드를
-//     주입받을 수 있습니다*/
-//
-//    @BeforeEach
-//    public void setup() {
-//        bbsController = new BbsController();
-//        mockMvc = MockMvcBuilders.standaloneSetup(bbsController).build();
-//    }
-//
-//    @Test
-//    @DisplayName("Get 컨트롤러 작동 확인")
-//    void getList() throws Exception {
-//        mockMvc.perform(get("/api/message"))
-//                .andExpect(status().isOk())
-//                .andExpect(content().string("Hello, World!"));
-//    }
-//}
\ No newline at end of file
Index: src/test/java/com/encore/bbs/mapper/testMapper.java
===================================================================
diff --git a/src/test/java/com/encore/bbs/mapper/testMapper.java b/src/test/java/com/encore/bbs/mapper/testMapper.java
deleted file mode 100644
--- a/src/test/java/com/encore/bbs/mapper/testMapper.java	
+++ /dev/null	
@@ -1,73 +0,0 @@
-package com.encore.bbs.mapper;
-
-import com.encore.bbs.board.mapper.BbsMapper;
-import com.encore.bbs.board.service.BbsService;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-
-import static org.junit.jupiter.api.Assertions.*;
-
-//@SpringBootTest
-//public class testMapper {
-//    @Autowired
-//    private BbsMapper bbsMapper;
-//    @Autowired
-//    private BbsService bbsService;
-
-//    @Test
-//    @DisplayName("DB연결 후 조회 테스트")
-//    public void testBbsDetail(){
-//        try {
-//            List<BbsDTO> bbsDetail = bbsMapper.selectBbsDetail(bbsId);
-//            assertNotNull(bbsDetail);
-//            assertTrue(bbsDetail.size() > 0);
-//            BbsDTO firstBbs = bbsDetail.get(0);
-//            assertNotNull(firstBbs);
-//            assertTrue(firstBbs.getBbsId() > 0);
-//        } catch (Exception e) {
-//            e.printStackTrace();
-//        }
-//    }
-
-//    @Test
-//    public void testMapping() {
-//        assertThrows(Exception.class, () -> {
-//            bbsMapper.selectBbsDetail();
-//            }, "Expected BindingException was not thrown.");
-//    }
-
-//    @Test
-//    @DisplayName("대륙 DB연결 후 조회 테스트")
-//    public void testContinent(){
-//            Long continentId = 1L;
-//
-//            ContinentDto continentDto = bbsMapper.getContinentByCountryId(continentId);
-//
-//            assertNotNull(continentDto);
-//            assertEquals(continentId, continentDto.getContinentId());
-//        }
-//    @Test
-//    @DisplayName("매퍼 붙는 테스트")
-//    public void testMapping() {
-//        Long continentId = 1L;
-//
-//        assertThrows(Exception.class, () -> {
-//            bbsMapper.getContinentByCountryId(continentId);
-//            }, "Expected BindingException was not thrown.");
-//    }
-//
-//    @Test
-//    @DisplayName("서비스 테스트")
-//    public void testService(){
-//        Long continentId = 1L;
-//
-//        ContinentDto resultContinentDto = bbsService.getContinentByCountryId(continentId);
-//
-//        assertNotNull(resultContinentDto);
-//        assertEquals(continentId, resultContinentDto.getContinentId());
-//        assertNotNull(resultContinentDto.getContinentName());
-//    }
-//    }
-
